go/protocol/chat1/extras.go

```diff
@@ -1683,6 +1683,35 @@ func (i InboxUIItem) GetConvID() ConversationID {
 	return ConversationID(bConvID)
 }
 
+func (i InboxUIItem) ExportToSummary() (s ConvSummary) {
```

Comment:
I'd rather not have this kind of stuff in protocol, I know it makes the call sites more convenient, but it just strikes me as wrong. I think it works best if stuff in extras is just simple, and more complicated things (like this) can go into `utils`.